lib-api/src/main/java/org/ergoplatform/appkit/IErgoBox.java

```diff
@@ -0,0 +1,34 @@
+package org.ergoplatform.appkit;
+
+import java.util.List;
+
+import sigmastate.Values;
+
+public interface IErgoBox {
```

Comment:
Better TxBox or TransactionBox.
Please add JavaDoc including the motivation for adding interface.